app/controllers/products_controller.rb

```diff
@@ -1,8 +1,34 @@
 class ProductsController < ApplicationController
 	before_action :find_product, only: :show
-
 	def index
-		@products = Product.all
+
+		if params["classify"] == "mans"
```

Comment:
@products = Product.send(params["classify"])

Reply:
[code](byebug) params[:classify].persent?
*** NoMethodError Exception: undefined method `persent?' for "mans":String
Did you mean?  present?[/code]